src/components/Gauge/Gauge.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { ContentPlaceholder } from '../ContentPlaceholder';
+import { Context, GAUGE_RADIUS } from '../../constants';
+import styles from './Gauge.scss';
+
+const { block, elem } = bem('Gauge', styles);
+
+interface Props {
+    /** The gauge context (e.g. brand, primary, bad, good etc. - defaults to brand) */
+    context?: Context;
+    /** Defines if progress bar is in loading state */
+    isProgressLoading?: boolean;
+    /** Defines if content part is in loading state */
+    isContentLoading?: boolean;
+    /** Percentage of the progress bar to be filled */
+    percentage?: number;
+    /** Content component */
+    children: NotEmptyReactNode;
+    /** Metric component */
+    metric?: SingleReactNode;
+    /** Note component */
+    note?: SingleReactNode;
+}
+
+export const Gauge: React.FC<Props> = (props) => {
+    const {
+        children,
+        percentage = 0,
+        context,
+        note,
+        metric,
+        isProgressLoading,
+        isContentLoading,
+        ...rest
+    } = props;
+
+    const percentageAdjusted = Math.max(0, Math.min(100, percentage)) / 100;
+    const progress = isProgressLoading ? 0 : percentageAdjusted;
+    const circumference = 2 * Math.PI * GAUGE_RADIUS;
+    const circumferenceHalf = circumference / 2;
+
+    return (
+        <div {...rest} {...block(props)}>
+            <svg {...elem('svg', props)} width="100%">
+                <linearGradient id={`Gauge__gradient--${context}`}>
+                    <stop className={`Gauge__gradientStart--${context}`} offset="50%" />
+                    <stop className={`Gauge__gradientEnd--${context}`} offset="100%" />
+                </linearGradient>
+                <circle
+                    {...elem('circleBackground', props)}
+                    r={GAUGE_RADIUS}
+                    cx="50%"
+                    cy="25%"
+                    style={{ strokeDasharray: `${circumferenceHalf} ${circumference}` }}
+                />
+                <circle
+                    {...elem('circleForeground', props)}
+                    r={GAUGE_RADIUS}
+                    cx="50%"
+                    cy="25%"
+                    style={{ strokeDasharray: `${progress * circumferenceHalf} ${circumference}` }}
+                    stroke={`url(#Gauge__gradient--${context})`}
+                />
+            </svg>
+            <div {...elem('contentWrapper', props)}>
+                {isContentLoading ? (
+                    <ContentPlaceholder height={28} {...elem('contentPlaceholder', props)} />
+                ) : (
+                    <span {...elem('content', props)}>
+                        {children}
+                        {metric ? <span {...elem('metric', props)}>{metric}</span> : null}
```

Comment:
If `children` and `metric` are ReactNodes, then there is a good chance that wrapping them in `span` will create incorrect HTML.